src/main/java/Portfolio/BlueprintsManagement/application/service/ArchitecturalDrawingService.java

```diff
@@ -0,0 +1,47 @@
+package Portfolio.BlueprintsManagement.application.service;
+
+import Portfolio.BlueprintsManagement.domain.model.architecturalDrawing.ArchitecturalDrawing;
+import Portfolio.BlueprintsManagement.domain.repository.IArchitecturalDrawingRepository;
+import Portfolio.BlueprintsManagement.presentation.dto.request.architecturalDrawing.ArchitecturalDrawingRequest;
+import Portfolio.BlueprintsManagement.presentation.exception.customException.NotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ArchitecturalDrawingService {
+
+    private final IArchitecturalDrawingRepository architecturalDrawingRepository;
+
+    public List<ArchitecturalDrawing> getArchitecturalDrawings(String blueprintId) throws NotFoundException {
+        return architecturalDrawingRepository.getArchitecturalDrawingsByBlueprintId(blueprintId);
+    }
+
+    public ArchitecturalDrawing addArchitecturalDrawing(ArchitecturalDrawingRequest request) throws IOException {
```

Comment:
`@Transactional`アノテーションあった方が良さそうです。
`architecturalDrawingRepository.addArchitecturalDrawing(architecturalDrawing);` に成功しても、次の `Files.write(filePath, content);` に仮に失敗してしまった場合、データベースのデータがロールバックできないためです。

```suggestion
    @Transactional
    public ArchitecturalDrawing addArchitecturalDrawing(ArchitecturalDrawingRequest request) throws IOException {
```